cmd/util/fatal.go

```diff
@@ -14,16 +14,17 @@ import (
 var Fatal = fatalError
 
 func fatalError(cmd *cobra.Command, err error, code int) {
-	cmd.PrintErrln()
+	cmd.SetOut(os.Stdout)
+	cmd.Println()
```

Comment:
I don't think setting the writer here is necessary as we are already doing that in cli/root

https://github.com/bacalhau-project/bacalhau/blob/48dc1a709d6fc6456fc49e7bcd5ed0a2fb919e76/cmd/cli/root.go#L136-L137